policyengine_us/variables/gov/hhs/medicaid/income/tax_unit_medicaid_income_level.py

```diff
@@ -15,5 +16,14 @@ class tax_unit_medicaid_income_level(Variable):
 
```

Comment:
Please add these sources to the documentation above with comments explaining:

https://www.sos.state.co.us/CCR/GenerateRulePdf.do?ruleVersionId=11618&fileName=10%20CCR%202505-10%208.100

https://www.cms.gov/marketplace/technical-assistance-resources/special-populations-pregnant-women.pdf

Reply:
Let me know if you want me to format it differently.